web/src/main/java/org/cbioportal/genome_nexus/web/mixin/SignalPopulationStatsMixin.java

```diff
@@ -0,0 +1,23 @@
+package org.cbioportal.genome_nexus.web.mixin;
+
+import io.swagger.annotations.ApiModelProperty;
+
+public class SignalPopulationStatsMixin {
+    @ApiModelProperty("Impact")
+    private Double impact;
+
+    @ApiModelProperty("European")
```

Comment:
The name is referred from here: https://gnomad.broadinstitute.org/faq#how-are-population-names-abbreviated
Does SIGNAL have the same pattern?